docs/admin/kubeadm.md

```diff
@@ -27,7 +27,7 @@ following steps:
    dropping it in the cert directory (configured via `--cert-dir`, by default
    `/etc/kubernetes/pki`), this step is skipped.
 
-1. Outputting a kubeconfig file for the kubelet to use to connect to the API
```

Comment:
Two unrelated changes, not sure where to put:

(1) Can you copy and paste the following at the top of this file? This doc is kinda long so a table of contents would help.
```
* TOC
{:toc}
```

(2) I noticed that this is one of the only pages that drops the navigation sidebar---I looked into it, and to fix it, you need to change the following in `_data/reference.yml`:
replace 
```
- title: Kubeadm
  path: /docs/admin/kubeadm/
```
with 
```
- docs/admin/kubeadm.md
```
(The *.md is important since the sidebar nav does a loopy thing that needs to match the filename exactly)